### DISCUSSION TITLE
Showing arrow on dropdown, like a tooltip does

### DISCUSSION THREAD

**Discussion by Ericlm** [bootstrap-vue/bootstrap-vue#6897] [Q&A]
Hello, I'm looking for a way to show an arrow from a dropdown menu to the dropdown toggle, just like tooltips do. That is, the following, but for a dropdown:
<img width="193" alt="Capture d’écran 2022-01-16 à 15 27 16" src="https://user-images.githubusercontent.com/19361503/149664077-3f43f8a3-7c55-42d8-b370-f8b804ccb406.png">
 
I guess the best way is to pass custom popper options via the props `popper-opts` of the b-dropdown. However, I don't know anything about Popper JS, and I don't know how to achieve what I want.

**Comment by sidneygijzen** [comment]
One way would be a CSS triangle on a HTML pseudo element, e.g `dropdown-menu:before` or `dropdown-menu:after`. See [this page](https://css-tricks.com/the-shapes-of-css/#aa-triangle-up-shape) for some examples.
